tools/pd-ctl/pdctl/command/scheduler.go

```diff
@@ -34,9 +35,45 @@ func NewSchedulerCommand() *cobra.Command {
 	c.AddCommand(NewShowSchedulerCommand())
 	c.AddCommand(NewAddSchedulerCommand())
 	c.AddCommand(NewRemoveSchedulerCommand())
+	c.AddCommand(NewPauseSchedulerCommand())
+	c.AddCommand(NewResumeSchedulerCommand())
 	return c
 }
 
+// NewPauseSchedulerCommand returns a command to pause a scheduler.
+func NewPauseSchedulerCommand() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "pause <scheduler> <delay>",
+		Short: "pause a scheduler",
+		Run:   pauseOrResumeSchedulerCommandFunc,
+	}
+	return c
+}
+
+func pauseOrResumeSchedulerCommandFunc(cmd *cobra.Command, args []string) {
+	if len(args) != 2 && len(args) != 1 {
+		cmd.Usage()
+		return
+	}
+	path := schedulersPrefix + "/" + args[0]
+	input := make(map[string]int)
```

Comment:
https://play.golang.org/p/1VCw7y0aDJ2

I think you do not need to define input as `map[string]int`, just `map[string]interface{}` is OK. When you marshal it, the type would be inferred by reflecting. See the above link for details.

So, `postData` is not necessary as well.

Reply:
thanks, finish.